example.js

```diff
@@ -9,28 +9,34 @@ rosnodejs.initNode('/my_node')
   // EXP 1) Service Server
   let service = rosNode.advertiseService('/set_bool', SetBool,
     (req, resp) => {
-      console.log('Handling request! ' + JSON.stringify(req));
+      rosnodejs.log.info('Handling request! ' + JSON.stringify(req));
```

Comment:
In general, I am a fan of verbosity. And I do like this fact that it is super obvious the kind of logger you are using here. But I am wondering how easy would it be for a user to alias `rosnodejs.log.info` as something like `ros.info`? Not saying that we need to do it for them though, just curious.